daffodil-cli/src/it/scala/org/apache/daffodil/executing/TestCLIexecuting.scala

```diff
@@ -130,6 +139,8 @@ class TestCLIexecuting {
     try {
       shell.sendLine(String.format("%s test -l %s", Util.binPath, testTdmlFile))
       shell.expect(contains(output16))
+
+      Util.expectExitCode(ExitCode.TestError, shell)
```

Comment:
Some of these exit codes seem incorrect to me. For example, this test is just listing the tests in a tdml file. That should succeed, so I would expect this to have an exit code of ExitCode.Success. Feels like this test is checking for the wrong exit code and it's going to fail, or the CLI is returning a TestError when it should be returning a Success.